### DISCUSSION TITLE
genuine plea for help

### DISCUSSION THREAD

**Discussion by davidulu25** [tweepy/tweepy#2106] [Questions] [answered]
please I'm a new beginner and still figuring out how to send out tweets with my automated twitter account. It seems twitter is asking me to apply for elevated access. Is there any way I can access endpoints that allow me to tweet without applying for elevated access?

**Comment by Harmon758** [comment, marked as answer]
Yes, you can just use Free access.
https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api
https://twittercommunity.com/t/announcing-new-access-tiers-for-the-twitter-api/188728
